test/instrumentation/modules/mysql/pool-release-1.js

```diff
@@ -32,6 +32,7 @@ test('release connection prior to transaction', function (t) {
 function createPool (cb) {
   setup(function () {
     var pool = mysql.createPool({
+      host: process.env.MYSQL_HOST || 'localhost',
```

Comment:
No need to specify `localhost` as the fallback as it will happen automatically if `MYSQL_HOST` isn't set